target/linux/ipq806x/base-files/etc/board.d/02_network

```diff
@@ -55,6 +55,10 @@ zyxel,nbg6817)
 	ucidef_set_interface_macaddr "lan" "$(macaddr_add $hw_mac_addr 2)"
 	ucidef_set_interface_macaddr "wan" "$(macaddr_add $hw_mac_addr 3)"
 	;;
+askey,rt4230w)
+	ucidef_add_switch "switch0" \
+		"1:wan" "2:lan" "3:lan" "4:lan" "6@eth1" "5:lan" "0@eth0"
```

Comment:
If you don't care about the port order anyway, please put the ethX entries in a way that they separate lan from wan ports ...